lambda-eventbridge-sqs-lambda-cdk-python/app.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+import os
+import aws_cdk as cdk
+import json 
+
+from aws_cdk import (
+    Duration,
+    Stack,
+    aws_iam as iam,
+    aws_lambda as lambda_,    
+    aws_lambda_event_sources as event_sources,    
+    aws_sqs as sqs_,
+    aws_events as events,    
+    aws_events_targets as targets,
+    CfnOutput,
+)
+from aws_solutions_constructs.aws_sqs_lambda import SqsToLambda
+from constructs import Construct
+
+DIRNAME = os.path.dirname(__file__)
+
+
+class LambdaEventBridgeSQSLambda(Stack):
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        
+
+        # Iam role to invoke lambda
+        lambda_cfn_role = iam.Role(
+            self,
+            "CfnRole",
+            assumed_by=iam.ServicePrincipal("events.amazonaws.com"),
+        )
+        lambda_cfn_role.add_managed_policy(
+            iam.ManagedPolicy.from_aws_managed_policy_name("AWSLambdaExecute")
+        )
+
+        # lambda function for processing the incoming request from S3 bucket
+        lambda_function = lambda_.Function(
```

Comment:
I would explicitly define the function log groups and deletion policy so they are cleaned up during cdk destroy.

Reply:
done